notebooks/MeasurementOrFact_Analysis.ipynb

```diff
@@ -0,0 +1,2951 @@
+{
```

Comment:
<p>It may be worth expanding here. "We can safely assume that the NaNs in this data are intended to be 0. NaN often implies that the data wasn't recorded or the column isn't applicable for this row, but in this case it looks like the <code>0</code>s were entered as NaN."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>Noted! I will update the story here. I did actually mean the latter option - the columns didn't apply to this record.</p>